### PR TITLE
Fix warnings

### PR DESCRIPTION
**Pull request by gperciva**

### CODE REVIEW

File: lib/util/getfstype.c [hunk 1]
```diff
@@ -62,7 +62,7 @@
 struct ftyperec {
 	unsigned int f_type;
 	const char * typename;
-} ftypes[] = {
+} static ftypes[] = {
```
Comment (cperciva) [inline lib/util/getfstype.c]: Normally we put `static` before the type... is there some reason to do otherwise here?

**Comment by gperciva** [comment]
I was confused by the struct definition.  Updated with rebased PR, fixing the `static`.

File: lib/network/tsnetwork_select.c [hunk 2]
```diff
@@ -46,9 +46,6 @@ network_select(int blocking)
 		return (events_run());
 	}
```
Comment (cperciva) [inline lib/network/tsnetwork_select.c]: This should have `/* UNREACHABLE */` added, just for the sake of clarity.  Sorry I didn't notice this on the first pass.

**Comment by gperciva** [comment]
Updated PR, rebased onto current master, and adding `/*  UNREACHABLE */`

**Pull request merged**